src/cheri_insts.sail

```diff
@@ -280,6 +303,26 @@ function clause execute(CSetFlags(cd, cs1, rs2)) = {
 }
 
 union clause ast = CToPtr : (regidx, regidx, regidx)
+/*!
+ * If the **tag** field of capability register *cs1* is not set then integer
+ * register *rd* is set to 0, otherwise integer register *rd* is set to
+ * *cs1*.**address** $-$ *cs2*.**base**.
+ *
+ * ## Exceptions
+ *
+ * An exception is raised if:
+ *   - *cs2*.**tag** is unset.
+ *   - *cs1*.**tag** is set and *cs1* is sealed.
+ *
+ * ## Notes
+ *
+ *   - *cs2* being sealed will not cause an exception to be raised. This is for
+ *     further study.
+ *
+ *   - [CGetAddr] similarly allows access to the sum of the base and offset of
```

Comment:
I think we can drop the base + offset bit? Also I don't really think this relates to cgetaddr, csub is much more similar.
Or maybe completely drop this MIPS note?

Reply:
Yeah I wasn't sure about it. I dropped a couple of others already, I'll just drop this too.